### ISSUE TITLE
Issue with FreeCAD model

### ISSUE THREAD

**Issue by rlavriv** [jlecomte/automated-telescope-dust-cover#2]
When I am trying to change globals in FreeCAD model to adjust to my OTA, I get issue with model recompile.

The issue seem to be with USB Connector hole sketch, which shows in a wrong plane:
![Screenshot_20230501_155454](https://user-images.githubusercontent.com/19315295/235529569-432c4f9c-a8ad-40c9-8228-fcecc6d56c37.png)

Any ideas how to fix it?

FreeCAD 0.20.2 on Fedora 38

**Comment by jlecomte** [comment]
That is because of the topological naming problem, which is a huge issue in FreeCAD. See:

* https://wiki.freecad.org/Topological_naming_problem
*  https://ondsel.com/blog/freecad-topological-naming/

Unfortunately, I did not follow the best practices when designing this model, and therefore it is a victim of this issue.

Since I may upgrade my scope to a larger one, I may completely redesign this in FreeCAD, but this time doing it correctly (and also simplifying the design a bit)...

**Comment by rlavriv** [comment]
Is there absolutely no way to salvage it?

**Comment by jlecomte** [comment]
I don't think so. Now, I'm hearing that FreeCAD 0.22 will have mostly solved the topological naming issue, so it might be worth trying then. best of luck!
